core/domain/user_services.py

```diff
@@ -79,11 +78,12 @@ def __init__(
             last_agreed_to_terms=None, last_started_state_editor_tutorial=None,
             last_logged_in=None, last_created_an_exploration=None,
             last_edited_an_exploration=None, profile_picture_data_url=None,
-            default_dashboard=None, user_bio='', subject_interests=None,
-            first_contribution_msec=None, preferred_language_codes=None,
-            preferred_site_language_code=None,
-            preferred_audio_language_code=None
-    ):
+            default_dashboard=None,
+            creator_dashboard_display=
```

Comment:
Would better to right this in this fashion - 
```
creator_dashboard_display=(
feconf.CREATOR_DASHBOARD_DISPLAY_PREF_CARD)
```

Reply:
Done